source/test/F0.Minesweeper.Components.Tests/Logic/Cell/CellStatusManagerTests.cs

```diff
@@ -93,8 +164,8 @@ private static CellStatusManager CreateCellStatusManager(CellStatusType currentS
 		{
 			var componentForTest = new CellStatusManager();
 
-			FieldInfo? currentStatusField = typeof(CellStatusManager).GetField("currentStatus", BindingFlags.NonPublic | BindingFlags.Instance);
-			currentStatusField?.SetValue(componentForTest, currentStatus);
+			PropertyInfo? currentStatusProperty = typeof(CellStatusManager).GetProperty(nameof(CellStatusManager.CurrentStatus));
+			currentStatusProperty?.SetValue(componentForTest, currentStatus, BindingFlags.NonPublic | BindingFlags.Instance, null, null, null);
```

Comment:
I am unsure about the _null-conditional_ `SetValue` here: what should happen if the property is not found? Should it be silently ignored?
I believe it would be better to have the null-reference-exception if the `CurrentStatus` is not found, to indicate an error here. I would use a _null-forgiving_ operator here instead.

Reply:
Changed it to *null-forgiving* operator